Tests/SentryTests/UIRedactBuilderTests.swift

```diff
@@ -394,6 +405,32 @@ class UIRedactBuilderTests: XCTestCase {
             XCTAssertTrue(sut.containsIgnoreClass(element), "\(element) not found")
         }
     }
+    
+    func testLayerIsNotFullyTransparentRedacted() {
+        let sut = getSut()
+        let view = CustomVisibilityView(frame: CGRect(x: 20, y: 20, width: 40, height: 40))
+        view.alpha = 0
+        view.sentryReplayMask()
+        
+        view.backgroundColor = .purple
+        rootView.addSubview(view)
+        
+        let result = sut.redactRegionsFor(view: rootView)
+        XCTAssertEqual(result.count, 1)
```

Comment:
`m`: Does it make sense to assert for more than only the count?

Reply:
I think so, because before this PR it was 0.
It means if the CustomView layer is animating its opacity, we still should mask it.